auth/session_test.go

```diff
@@ -0,0 +1,168 @@
+package auth
+
+import (
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/couchbase/sync_gateway/base"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	Username         = "Alice"
+	TwoHours         = 2 * time.Hour
+	TwentyFourHours  = 24 * time.Hour
+	NoExpiry         = 0
+	NegativeTtl      = -1
+	NoTtl            = 0
+	ErrMsgIllegalTtl = "400 Invalid session time-to-live"
+	DefaultURL       = "http://localhost/"
+)
+
+func TestDocIDForSession(t *testing.T) {
```

Comment:
Sure Adam, I did check this. The `DocIDForSession` function is already covered in `TestDeleteSession` and we can remove it. But just kept it with changing the assert logic to `assert.Contains(t, docIDForSession, sessionID)` instead of mocking up the actual value inside the test itself. By that way it won't cause any harm even if we change the `DocIDForSession` logic in future.